### DISCUSSION TITLE
目前码支付和YunGouOS接口是无法使用吗？

### DISCUSSION THREAD

**Discussion by Ecalose** [Baiyuetribe/kamiFaka#40] [Bug反馈] [answered]
貌似码支付测试时付完钱后没反应，YunGouOS直接无法调用

**Comment by Baiyuetribe** [comment, marked as answer]
码支付回调不正常，无法使用；YunGouOS的需要有人参与调试，当时别人提供我就对接了，没有实际账号进行测试。如果你需要该接口，可联系我进行调试。

**Reply by Ecalose** [reply]
okk，怎么联系呢？

**Reply by Ecalose** [reply]
tg找你还是？

**Reply by Baiyuetribe** [reply]
已修复YunGouOS接口https://github.com/Baiyuetribe/kamiFaka/commit/9a922618b97d16ebf28679477c8eeb473d2858a1

**Reply by Ecalose** [reply]
老哥，貌似还有点问题，在手机端唤醒后无法直接跳转支付，而且微信禁止截图扫码,或许应该对接h5？
补个文档[https://open.pay.yungouos.com/#/api/api/pay/wxpay/wapPay](url)

**Reply by Baiyuetribe** [reply]
给你说的清清楚楚的，微信唤醒需要其他的。

**Reply by n0ah37** [reply]
你好，那请问码支付什么时候修复呢，谢谢

**Reply by Baiyuetribe** [reply]
因码支付接口限制，无法修复，该接口会弃用，后续会增加V免签
